cmd/hub/Dockerfile

```diff
@@ -16,8 +17,9 @@ RUN yarn install
 RUN yarn build
 # Generate a yarn.lock version that does not contain the dev dependencies
 RUN cp -R /web/node_modules /web/package.json /web/yarn.lock /tmp
-RUN cd /tmp && cat package.json | jq 'del(.devDependencies)' > tmp && mv tmp package.json
-RUN cd /tmp && yarn install
+WORKDIR /tmp
+RUN cat package.json | jq 'del(.devDependencies)' > tmp && mv tmp package.json
```

Comment:
IIRC that was still including the dev deps in the `yarn.lock` file, but I'd need to double check to be sure.

The `yarn.lock` file generated on those lines is copied to the final `hub` image so that Trivy can scan it for security vulnerabilities, it's not used for anything else.